gamefile_reader.html

```diff
@@ -925,6 +927,8 @@
 			region_text += `  Volume: ${region_volume}\n`;
 			region_text += `  Pitch bytes: ${region_pitch_a} ${region_pitch_b}\n`;
 			region_text += `  Sample data offset: 0x${region_sample_offset.toString(16)}\n`;
+			region_text += `  Note start: 0x${region_note_start.toString(10)}\n`;
+			region_text += `  Note end: 0x${region_note_end.toString(10)}\n`;

```

Comment:
this information is redundant, its already included a few lines above... and the 0x is misleading.

Reply:
Ooo sorry, I fixed those.